pkg/cmd/release/download/download_test.go

```diff
@@ -391,6 +391,9 @@ func Test_downloadRun(t *testing.T) {
 }
 
 func Test_downloadRun_cloberAndSkip(t *testing.T) {
+	oldAssetContents := "older copy to be clobbered"
```

Comment:
Might possibly be worth making it clear in the test that this 'old' contents needs to be larger than the new download for the test to be valid. Incase someone modifies 'somedata' in the future.

Reply:
Thanks! Good thought.